src/mmhomepagepanel.cpp

```diff
@@ -825,14 +823,14 @@ const wxString mmHomePagePanel::displayIncomeVsExpenses()
     json_writer.Int(scaleStepWidth);
     json_writer.EndObject();
 
-    wxLogDebug("======= mmHomePagePanel::displayIncomeVsExpenses =======");
+    wxLogDebug("======= mmHomePagePanel::getIncomeVsExpensesJSON =======");
```

Comment:
can we use __func__ here?

Reply:
[`__func__` identifier](https://stackoverflow.com/a/4384825) intended for such use